keycloak4s-admin/src/main/scala/com/fullfacing/keycloak4s.admin/services/package.scala

```diff
@@ -15,6 +18,18 @@ package object services {
   type Path = ImmutableSeq[String]
   implicit def uuidToString: UUID => String = _.toString
 
+  def extractStringFromResponse(response: Either[KeycloakError, Headers]): Either[KeycloakError, String] = response.flatMap { headers =>
+    headers
+      .get("Location")
+      .map(location => location.split('/').last)
+      .toRight(Exceptions.ID_NOT_FOUND)
+  }
+
+  def extractUuidFromResponse(response: Either[KeycloakError, Headers]): Either[KeycloakError, UUID] = response.flatMap { headers =>
```

Comment:
The name of the function is a bit overloaded and it's on the package level which indicates a high level function while it's specific to the Location header of a response ;) Either generalize it or make it more local/less accessible